src/plugin.ts

```diff
@@ -30,35 +29,28 @@ export default function pluginWithTasks(
   overrideConfig?: AppOptions,
 ): ExtendedCypressConfig {
   // Only initialize admin instance if it hasn't already been initialized
-  if (adminInstance.apps && adminInstance.apps.length === 0) {
+  if (adminInstance.apps?.length === 0) {
```

Comment:
Ah, I thought there might have been a reason to not use it, my wishful thinking was just betting on it being old code or something like that.
Imo, it would not be worth requiring a higher min version of Cypress if that is the case.

Reply:
No worries - I often forget the exact limitation, so I think it would be worth making a note in the README around this and including exact version.

In the future we may want to build with esbuild or similar so that codebase can use newest functionality without worrying about backwards compatibility